lang/python/django/Makefile

```diff
@@ -8,11 +8,11 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=django
-PKG_VERSION:=4.0.7
+PKG_VERSION:=4.0.8
 PKG_RELEASE:=$(AUTORELEASE)
```

Comment:
we'd need to change this to `PKG_RELEASE:=1`

now :)

Reply:
as a general topic, there is a discussion to drop this AUTORELEASE feature
so, we will slowly drop it i guess

Reply:
done.